src/openapi/openapi.yml

```diff
@@ -439,9 +478,22 @@ components:
       required: true
       schema:
         $ref: '#/components/schemas/ID'
+    policy_search_term:
+      name: search_term
+      in: path
+      description: ID of the tower
```

Comment:
This might have been copied from something else?

Reply:
Thanks, Brian! I've just removed that specific parameter as it's no longer needed after the change to the search functionality.